bin/generate.py

```diff
@@ -456,16 +456,16 @@ def __init__(self, problem, generator_config, key, name: str, yaml, parent, coun
             # checks
             if not any(x in yaml for x in ['generate', 'copy', 'in', 'interaction']):
                 self.parse_error = 'Testcase requires at least one key in "generate", "copy", "in", "interaction". Skipping.'
-                config.n_error += 1
+                generator_config.n_parse_error += 1
```

Comment:
we may parse the yaml but dont run the generators (for example for bt stat) and than it would be weird if we dont display errors but exit with non zero exit code